### ISSUE TITLE
AG WG Review: 2.4.10 Section Headings (Level AAA) WCAG2ICT Guidance

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#917]
## Summary
Review of proposed WCAG2ICT guidance for the Level AAA WCAG SC: 2.4.10 Section Headings, as incorporated into the Editor's draft. 

## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  
* Level A and AA SCs can be found in the [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md) file.
* Level AAA SCs can be found in the [comments-on-level-AAA-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-on-level-AAA-success-criteria.md)
----

## Background information 

Once finalized, this new WCAG2ICT Note will build on the previous version by also covering WCAG 2.2 level AAA success criteria (currently only level A and AA criteria are covered.  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

## Content for Review

* Section [Applying SC 2.4.10 Section Headings to non-web documents and non-web software](https://w3c.github.io/wcag2ict/#applying-sc-2-4-10-section-headings-to-non-web-documents-and-non-web-software).  

## Reference Material
In case you need them, links to various resources:

* [2.4.10 Section Headings](https://www.w3.org/TR/WCAG22/#section-headings) SC text in WCAG 2.2
* [Intent section of Understanding 2.4.10 Section Headings](https://www.w3.org/WAI/WCAG22/Understanding/section-headings#intent)

**Comment by maryjom** [comment]
I think the WCAG2ICT Note 1 from 2.4.6 isn't directly applicable to SC 2.4.10 Section Headings. 2.4.10 is about requiring you to have headings for sections of content within writing - not user interface components (per the WCAG 2.4.10 Note 2). WCAG Note 1 explicitly states that this SC "covers sections within writing, not [user interface components](https://www.w3.org/TR/WCAG22/#dfn-user-interface-components)" and the current WCAG2ICT Note 1 says "headings and labels are used to describe sections of content **and controls** respectively" which seems to contradict the WCAG note (emphasis added by me). WCAG2ICT note 1 is also focusing on meaningful headings which is required by SC 2.4.6 Headings and Labels and not SC 2.4.10.

The WCAG2ICT notes should be geared more around the fact that one would need to provide section headings, where appropriate, for non-web software and non-web documents. There may not always be sections of written content, but when there is, 2.4.10 requires that headings are provided. SC 2.4.6 then requires the heading label or descriptor be meaningful. 

Consider providing examples of where 2.4.10 would require headings be created could be helpful (e.g. in embedded help or lengthy instructions for non-web software and in long documents where there are distinctly different parts or topics in the written document content. 

Also of note is that plain text documents are unlikely to be able to identify headings in any kind of structural, programmatic way, but could visually set apart text that could act as a heading.
